apps/climatemappedafrica/src/payload/blocks/DataVisualisationGuide.js

```diff
@@ -0,0 +1,32 @@
+import image from "../fields/image";
+
+const DataVisualisationGuide = {
+  slug: "data-visualisation-guide",
+  imageURL: "/images/cms/blocks/data-visualisation-guide.png",
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
```

Comment:
```suggestion
      required: true,
      localized: true,
```

Since other fields/collections already have `localized`, lets keep it.